src/desktop/apps/auction/actions/artworkBrowser.js

```diff
@@ -374,6 +374,7 @@ function formatDataForMPv2(saleArtworksConnection) {
     .map(saleArtwork => {
       const props = {
         ...saleArtwork,
+        ...saleArtwork.saleArtwork,
         ...saleArtwork.sale_artwork,
```

Comment:
Is this spreading confusing to people? Values from all three are needed in the component, so it's the easiest way to make sure `saleArtwork` has everything we need.

Reply:
not confusing, its mainly just flattening things out